alvr/server_io/src/openvrpaths.rs

```diff
@@ -27,20 +27,25 @@ fn openvr_source_file_path() -> Result<PathBuf> {
 }
 
 pub fn steamvr_settings_file_path() -> Result<PathBuf> {
-    let path = if cfg!(windows) {
+    if cfg!(windows) {
         // N.B. if ever implementing this: given Steam can be installed on another
         // drive, etc., this should probably start by looking at Windows registry keys.
         bail!("Not implemented for Windows.") // Original motive for implementation had little reason for Windows.
-    } else {
-        dirs::data_dir()
     }
-    .to_any()?
-    .join("Steam/config/steamvr.vrsettings");
 
-    if path.exists() {
-        Ok(path)
+    let steam_dir = steamlocate::SteamDir::locate()?;
+    let steamvr_vrsettings_path = steam_dir.path().join("config/steamvr.vrsettings");
```

Comment:
These don't need to be two distinct variables, you can just chain it

Reply:
old habits...